libs/shared/utils/src/getPaths/restrictedSlugs.ts

```diff
@@ -1,10 +1,12 @@
+// These are all of the Storyblok slugs that should not match pages/[slug]
+// because they are matched to other pages. For example, Labs 'blog' is matched
+// to apps/labs/pages/blog/index.tsx.
```

Comment:
```suggestion
// to apps/labs/pages/blog/index.tsx.
// Note that all these names will be excluded from pages/[slug]
// on *both* sites, since this is shared code.
```